src/Command/ModelCommand.php

```diff
@@ -483,7 +483,7 @@ public function getDisplayField(\Cake\ORM\Table $model, \Cake\Console\Arguments
             return $args->getOption('display-field');
         }
 
-        return $model->getDisplayField();
+        return (string)$model->getDisplayField();
```

Comment:
Might be better to change the method's return type hint to allow returning `null` instead. So `: string` to `: ?string`.